spec/granite/form/model/attributes/reflections/base_spec.rb

```diff
@@ -39,18 +41,7 @@ def reflection(options = {})
     let(:reflection) { SomeScope::Borogoves.new(:field, type: Object) }
     let(:owner) { Owner.new }
 
-    specify { expect(reflection.build_attribute(owner, nil)).to be_a(Granite::Form::Model::Attributes::Borogoves) }
-    specify { expect(reflection.build_attribute(owner, nil).name).to eq('field') }
-    specify { expect(reflection.build_attribute(owner, nil).owner).to eq(owner) }
-  end
-
-  describe '#type' do
-    before { stub_class(:dummy, String) }
-
-    specify { expect { reflection.type }.to raise_error('Type is not specified for `field`') }
-    specify { expect(reflection(type: String).type).to eq(String) }
-    specify { expect(reflection(type: :string).type).to eq(String) }
-    specify { expect(reflection(type: Dummy).type).to eq(Dummy) }
-    specify { expect { reflection(type: :blabla).type }.to raise_error NameError }
```

Comment:
Moved with slight modifications to `spec/granite/form/model/attributes/reflections/base/build_type_definition_spec.rb`